packages/react/src/themes/teams/components/Embed/embedVariables.ts

```diff
@@ -13,4 +14,5 @@ export default (siteVariables): EmbedVariables => ({
   width: undefined,
   height: undefined,
   focusBorderColor: siteVariables.colors.brand[500],
+  backgroundColor: siteVariables.colorScheme.brand.background,
```

Comment:
Why this?
Now when you play a video, because of the delay, there is a purple rectangle displayed.

Plus embed's `span` is 5px higher than the iframe, causing purple line to be displayed bellow the iframe:
![image](https://user-images.githubusercontent.com/9615899/62868694-80d6c600-bd16-11e9-96b1-0fb3b583ea0e.png)

To address this I would create a separate issue.

Reply:
https://stackoverflow.com/questions/21025319/iframe-creates-extra-space-below

Reply:
This was a poor attempt to resolve issue #1586. I have reverted it, let's address it by another PR in the future.